databricks/cli
# feat: support symlink resolution to one level

## Changes

This supports symlink files to be uploaded provided it can resolve the link. For the golang API this means the link points directly do regular file. If the symlink points to another symlink it will fail and databricks will act exactly as it does today and ignore the file.

This is largely because I want to write a bazel rule that uses databricks and without this support it's close to impossible as bazel heavily uses symlinks. 

## Tests

1. Created a symlink and ran `databrick bundle deploy` and see it's uploaded to the `files` folder. 
2. Create a symlink that points to another symlink and see that no error is thrown.
3. Create symlinks that reference each other and see that no error is thrown.
